rosbag2_storage_mcap/test/rosbag2_storage_mcap/test_message_definition_cache.cpp

```diff
@@ -151,10 +151,14 @@ TEST(test_message_definition_cache, get_service_message_definitions)
 TEST(test_local_message_definition_source, throw_definition_not_found_for_unknown_msg)
 {
   MessageDefinitionCache source;
-  ASSERT_THROW({ source.get_full_text("rosbag2_test_msgdefs/msg/UnknownMessage"); },
-               rosbag2_storage_mcap::internal::DefinitionNotFoundError);
```

Comment:
https://ci.ros2.org/job/ci_linux-rhel/1814/testReport/ can be reproducible with RHEL CI.

and this PR fixes the format aligned with rolling one.